webpack/scenes/ModuleStreams/Details/Profiles/ProfileRpmsCellFormatter.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from 'patternfly-react';
+
+import './ProfileRpmsCellFormatter.scss';
+
+class ProfileRpmsCellFormatter extends Component {
+  constructor(props) {
+    super(props);
+    this.minAmount = 10;
+    this.state = {
+      expanded: false,
+      showAmount: this.minAmount,
+    };
+  }
+
+  onClick = () => {
+    const { rpms } = this.props;
+
+    this.setState(state => ({
+      expanded: !state.expanded,
+      showAmount: !state.expanded ? rpms.length : this.minAmount,
```

Comment:
Does rpms data come from the redux store?
If it does, I tend to prefer to use redux store entirely ( with a selector for derived data) rather then the inner state.
 We had too many bugs because a combination between redux store and inner state.
@sharvit , any thoughts on this?

Reply:
@sharvit and I discussed about convention to use only redux state, which simplify the data flow, and therefore would be easier to understand and maintain. 
I'm fine with this minor use of inner state for now, at least until we will have such a convention.